packages/landing/app/page.tsx

```diff
@@ -76,7 +77,7 @@ export default function Home() {
           <div className="max-w-xl px-6 space-y-2 md:pb-28">
             <HeadingText size={4}>Using Stackly is super easy</HeadingText>
             <HeadingText weight="regular" className="text-em-med">
-              Create a stack (aka. recurring buy) in 3 steps.
+              Create a stack (aka. recurring buys) in 3 steps.
```

Comment:
```suggestion
              Create a stack (aka. recurring swaps) in 3 steps.
```

Reply:
Also "aka" doesn't have a dot at the end.

Create a stack (aka recurring buys/swaps) in 3 steps.

Reply:
All of the following are accepted AFAIK:
* `A.K.A`
* `AKA.`
* `AKA`
* `a.k.a`
* `aka.`
* `aka`

Reply:
Quick search shown me `aka` is the one. But I don't mind, anyone is fine if users understand.